src/state/experiments.js

```diff
@@ -3,9 +3,20 @@ import { add as addErrors } from './errors';
 
 /* reducer */
 
+export const UPLOAD_RES_NONE = 'ca/experiments/UPLOAD_RES_NONE';
+export const UPLOAD_RES_SUCCESS = 'ca/experiments/UPLOAD_RES_SUCCESS';
+export const UPLOAD_RES_FAILURE = 'ca/experiments/UPLOAD_RES_FAILURE';
+
 export const initialState = {
   loading: true,
   error: null,
+  createInProgress: false,
+  updateInProgress: false,
+  uploadInProgress: false,
+  uploadResult: UPLOAD_RES_NONE,
+
+  editModalShow: false,
+  editModalExpID: null,
```

Comment:
could you please keep names consistent with all other js files?
In js in general and in code-annotation frontend camelCase is a standard.

Reply:
I must be missing something, sorry, can you paste me the exact change you have in mind?

Reply:
`editModalExpID` -> editModalExpId`. Just because it's `Id` not `ID` everywhere.